source/ID/concordium-did.rst

```diff
@@ -0,0 +1,546 @@
+      "Concordium DID URI"
+    ],
+    "id": "did:ccd:NET:acc:ADDR",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:NET:acc:ADDR#acc-1",
+        "controller": "did:ccd:NET:acc:ADDR",
+        "type": "VerifiableCondition2021",
+        "blockchainAccountId": "ADDR",
+        "threshold": "T",
+        "conditionThreshold": [
+          {
+            "verificationMethod": [
+              {
+                "id": "did:ccd:NET:acc:ADDR#credential-1",
+                "controller": "did:ccd:NET:acc:ADDR",
+                "type": "VerifiableCondition2021",
+                "threshold": "R_1",
+                "conditionThreshold": [
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-1-1",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-1",
+                    "publicKeyMultibase": "XX"
+                  },
+                  "...",
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-N_1-1",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-1",
+                    "publicKeyMultibase": "YY"
+                  }
+                ]
+              }
+            ]
+          },
+          "...",
+          {
+            "verificationMethod": [
+              {
+                "id": "did:ccd:NET:acc:ADDR#credential-M",
+                "controller": "did:ccd:NET:acc:ADDR",
+                "type": "VerifiableCondition2021",
+                "threshold": "N",
+                "conditionThreshold": [
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-1-M",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-M",
+                    "publicKeyMultibase": "VV"
+                  },
+                  "...",
+                  {
+                    "id": "did:ccd:NET:acc:ADDR#key-N_M-M",
+                    "type": "Ed25519VerificationKey2020",
+                    "controller": "did:ccd:NET:acc:ADDR#credential-M",
+                    "publicKeyMultibase": "ZZ"
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ],
+    "authentication": [
+      "#acc-1"
+    ]
+  }
+
+
+Smart Contract Instance DID
+---------------------------
+
+The goal of the Smart Contract Instance DID is to provide meta-data about the contract instance.
+At the moment, the main piece of meta-data is the Concordium account that send the initialization transaction.
+
+The Smart Contract Instance DID Document MUST contain the following data:
+
+- ``@context`` - the attribute that expresses context information.
+- ``id`` - the DID of the smart contract instance.
+- ``owner`` - a DID of an account that initialized the contract instance represented as a JSON object containing fields ``id`` and ``account``.
```

Comment:
I don't think we should include this. If we do it should say "creator" instead.

Reply:
One reason to include this is that the issuers are account addresses that created an instance of a credential registry. I think there should be a way of referring to the contract owner.  We could use `creator` if this makes more sense.